slasher/rpc/server.go

```diff
@@ -104,7 +104,7 @@ func (ss *Server) IsSlashableAttestation(ctx context.Context, req *ethpb.Indexed
 }
 
 // IsSlashableBlock returns an proposer slashing if the block submitted
-// is a double proposal.
+// is a double proposal(no db update is being done).
```

Comment:
```suggestion
// is a double proposal (no db update is being done).
```

Reply:
Also, this comment could be misleading, `DetectDoubleProposals` saves all blocks passed into it, is does update the DB.

Reply:
good catch . the right comment for the wrong function

Reply:
fixed